### ISSUE TITLE
Cannot reference `include/opt.zig` without copying to `src`

### ISSUE THREAD

**Issue by zmenciso** [pbui-project/pbui-main#44]
**Describe the bug**
Cannot find out how to include the `opt.zig` file with the `@import` syntax, since Zig thinks the file is out of the scope of the package.

**Desktop (please complete the following information):**
 - OS: Arch Linux 5.5.2-zen
 - Zig 0.5.0+ab4ea5d3c

**Comment by squeek502** [comment]
You might be looking for [`LibExeObjStep.addPackagePath`](https://github.com/ziglang/zig/blob/7f975bf09f4e6e81d68c9a573ac2e31b997b5816/lib/std/build.zig#L1700) used in your `build.zig` (i.e. `exe.addPackagePath("opt", "include/opt.zig")` would make `@import("opt")` work for that target).

There's a minimal example of this sort of thing [here](https://github.com/ziglang/zig/tree/8624379c71d79f67032d7ff2fd71642c84c79621/test/standalone/pkg_import).

**Comment by squeek502** [comment]
Why was this closed?

**Comment by sambattalio** [comment]
Oh oops, I thought it was fixed but I see what you are saying now.

**Comment by zmenciso** [comment]
Yup, this will require some very slight modification to all our src files and the `build.exe`--pretty straightforward.  I might get around to this next week if I have the time.
